test/common_nn.py

```diff
@@ -302,12 +309,7 @@ def nll_loss_helper(input, target, weight, ignore_index):
     losses_and_weights = [nll_loss_helper(i, t, weight, ignore_index)
                           for i, t in zip(input, target)]
     losses, weights = zip(*losses_and_weights)
-    if isinstance(losses[0], Variable):
-        losses_tensor = torch.stack(losses).type_as(input)
-        if not torch._C._with_scalars():
-            losses_tensor = losses_tensor.squeeze(1)
-    else:
-        losses_tensor = torch.Tensor(losses).type_as(input)
+    losses_tensor = torch.Tensor(losses).type_as(input)
```

Comment:
can we use variable/tensor with dtype here?  (if it's easy for you, it might not be hooked up fully yet, I'd have to check).  I'd just rather avoid introducing more torch.Tensor calls.